Applications/Tizen_native/CustomShortcut/src/main.c

```diff
@@ -31,14 +31,32 @@ static int routes_to_(appdata_s *ad, const char *source) {
 }
 
 /**
- * @brief thread runner wrapper for adding back callback again
+ * @brief main thread runner wrapper for adding back callback again
  *
  * @param data
  */
-static void add_back_cb_(void *data) {
+static void notify_train_done(void *data) {
   appdata_s *ad = (appdata_s *)data;
+
+  char buf[256];
+
+  const char *source = "train_result";
+  sleep(1);
```

Comment:
what is the purpose of this sleep?

Reply:
After training is done, before moving to training done user can see the last epoch result a bit longer to queue user that it has been finished. This enables smoother transition

Reply:
Lets add a small comment here then.

Reply:
Done